### PR TITLE
UX-2133/Added Grafana and Kubernetes Dashboard links for capi clusters on capi …

### PR DESCRIPTION
**Pull request by kevinylee1991**
…cluster details page

I believe I have the links correct, but it's currently inaccessible, think it is a backend issue. Have posted in cluster-api channel about it.

Screen Shot 2022-09-07 at 5 10 10 PM

### CODE REVIEW

File: src/app/plugins/infrastructure/components/clusters/capi/actions.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export const listCapiClusters = capiClusterActions.add(
     .addDependency(DataKeys.AwsClusterRoleIdentities)
     .addDependency(DataKeys.AwsClusterStaticIdentities)
     .addDependency(DataKeys.CloudProviders),
+  // .addDependency(DataKeys.ClusterAddons),
```
Comment (patil-pratik-87) [inline src/app/plugins/infrastructure/components/clusters/capi/actions.ts:36]: @kevinylee1991 This was helpful for refreshing the cluster addons statuses when we refresh or poll. Is there a reason to not have this?

**Review by patil-pratik-87 — APPROVED** [review]

File: src/app/plugins/infrastructure/components/clusters/capi/actions.ts (hunk 1 above)
Comment (patil-pratik-87) [reply src/app/plugins/infrastructure/components/clusters/capi/actions.ts:38]: ohh okay.

**Review by patil-pratik-87 — APPROVED** [review]

File: src/app/plugins/infrastructure/components/clusters/capi/helpers.ts [hunk 2]
```diff
@@ -72,3 +75,18 @@ export const getIdentityResource = (
   }
   return null
 }
+
+export const getCapiClusterProxyEndpoint = (cluster) =>
+  `${sunpike.baseUrl}/apis/opt.pf9.io/v1beta1/namespaces/${cluster?.metadata?.namespace}/clusters/${cluster?.metadata?.name}/proxy`
+
+export const getCapiK8sDashboardLink = (cluster) => {
+  return `${window.location.origin}${getCapiClusterProxyEndpoint(
```
Comment (patil-pratik-87) [inline src/app/plugins/infrastructure/components/clusters/capi/helpers.ts:83]: @kevinylee1991 The change that Erwin has made needs the auth token in the form of cookies. This change will have to be made in the UI. 
For ref:- https://github.com/platform9/pf9-cluster-api/pull/270/files#diff-7fd0050121b6057009743f0050ecc35f6b97cbff89f0672e8ddca3973890d3e6R142

**Review by patil-pratik-87 — CHANGES_REQUESTED** [review]

File: src/app/plugins/infrastructure/components/clusters/capi/helpers.ts (hunk 2 above)
Comment (kevinylee1991) [reply src/app/plugins/infrastructure/components/clusters/capi/helpers.ts:86]: The auth token is already in the cookies (set by token2cookie api request) so we're all good

**Pull request merged**